src/libstd/fs.rs

```diff
@@ -756,11 +756,19 @@ impl AsInner<fs_imp::DirEntry> for DirEntry {
 /// guarantee that the file is immediately deleted (e.g. depending on
 /// platform, other open file descriptors may prevent immediate removal).
 ///
+/// # Platform behavior
+///
+/// This function currently corresponds to the `unlink` function on Unix
+/// and the `DeleteFile` function on Windows. Note that, this
+/// [may change in the future.][https://github.com/rust-lang/rust/pull/28613]
```

Comment:
Interesting. One of the things about this PR is that we need to figure out the standard way to encode this kind of thing, I guess a link to github would be reasonable.

Two nits here: 1: this needs to be `[]()` not `[][]`, and the period should be outside the link. However, we prefer the reference style links, like

```
/// [may change in the future][changes].
///
/// [changes]: url 
```

So that should be done, regardless.